### ISSUE TITLE
ApexLexer not found

### ISSUE THREAD

**Issue by Alfystar** [nawforce/apex-parser#25]
Hello, I try to run the tests, and boot in Java and JS, unfortunately, the code doesn't contain the ApexLexer class.

Can you please help me and update the readme?

**Comment by aaronhurst-google** [comment]
I just did a clean checkout and run into two issues:
* Java: if you're using JDK 11+, the javafx.util.Pair class is missing.  (I submitted a PR to remove it.)
* JS: The `antlr-patch` script `sed` command failed on my system. Changing the 2 copies of `-i ''` to just `-i` and it passed.

**Comment by nawforce** [comment]
I have resolved these in master so all should be good now, but let us know if you are still having troubles building from source.

**Comment by Alfystar** [comment]
I'm here, sorry for my delay.

The problem is about generation of code class using ANTLR4 grammar.

That's because it is my first iteration with it and I don't know which command I need to use...


I follow the instructions on the ANTLR4 site about installing and using ANTLR4 I generate the java class.

Unfortunately, I don't know where to put the java class generated to run the code, and about JS until now I didn't understand anything 😔.


I'm here to ask you for a tiny guide starting from the end of the ANTLR4 installation (or with the pip install command as command) and finish with the successful run of tests. 

I know ignorance is my fault, but unfortunately, it is hard to understand new thinking from zero.

**Comment by Alfystar** [comment]
This is my output follow build instruction:
```
09:49:20 ~/Documents/Salesforce/apex-parser (master)
└─▶ $ npm run build
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

> apex-parser-init@0.0.1 build
> npm run init && (cd npm; npm run build) && (cd jvm; mvn package)

npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

> apex-parser-init@0.0.1 init
> npm run init-npm && npm run init-jvm

npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.

> apex-parser-init@0.0.1 init-npm
> cp antlr/* npm/antlr && (cd npm; npm ci)

Impossibile trovare il percorso specification.
```

**Comment by aaronhurst-google** [comment]
Hi @Alfystar, `npm run build` will do all of the following: download the dependencies, compile the ANTLR grammar, compile the Java into a JAR, and run the tests. The result will be an archive `./jvm/target/apex-parser.jar` that you can put on the classpath for your JVM project and invoke with the code shown in the 'Example` section.

**Comment by Alfystar** [comment]
At the end of this day, yes I discover it

With my pull request I help other user that use "git bash" on Windows.
I unfortunately have to do this because in my company Linux is not accepted

**Comment by aaronhurst-google** [comment]
Ah, I had to work frequently with Cygwin in my last job. It was frequently an obstacle. Good luck. 😄

**Comment by nawforce** [comment]
Fixed by #32
